<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunWindow</class>
 <widget class="QMainWindow" name="RunWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>181</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="load_run_button">
       <property name="text">
        <string>Load Run parameters</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="start_run_button">
       <property name="text">
        <string>Start Run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stop_run_button">
       <property name="text">
        <string>Stop Run</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>260</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="post_analysis_check">
       <property name="text">
        <string>Post analysis</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="post_save_button">
       <property name="text">
        <string>Post Save directory</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>303</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
